obcsim/RS232.cpp

```diff
@@ -52,6 +52,24 @@ void RS_read(msp_link_t *lnk)
         Serial.println("--------------------------------------");
       }
       break;
+    case CMD_SEND_NVM_CITIROC_CONF:
+      Serial.println("CMD_SEND_NVM_CITIROC_CONF received");
+      len = 144;
+      if (fill_commanddata(len)) {
+        Serial.println("-------- Invoking SEND_NVM_CITI_CONF -----");
+        invoke_send(lnk, MSP_OP_SEND_NVM_CITI_CONF, commanddata, len, BYTES);
+        Serial.println("--------------------------------------");
+      }
+      break;
+    case CMD_SELECT_NVM_CITIROC_CONF:
+      Serial.println("CMD_SELECT_NVM_CITIROC_CONF received");
+      len = 1;
+      if (fill_commanddata(len)) {
+        Serial.println("-------- Invoking SELECT_NVM_CITIROC_CONF -----");
+        invoke_send(lnk, MSP_OP_SELECT_NVM_CITI_CONF, commanddata, len, BYTES);
+        Serial.println("--------------------------------------");
```

Comment:
Same comment as above for `SEND_NVM_CITIROC_CONF`.

Tip (maybe obvious, but hey): You can use the column of the print above to align where the dashes should end up.